### DISCUSSION TITLE
最新版alist cpu使用暴增

### DISCUSSION THREAD

**Discussion by bebestmaple** [AlistGo/alist#1652] [General]
docker 最新版

`2022/09/12 08:02:10.680798 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:02:44.442980 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1

2022/09/12 08:02:46.018365 ERROR RESTY Get "https://graph.microsoft.com/v1.0/me/drive/root:/images/xxxxxxxxxxxx:/children?$expand=thumbnails": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1

2022/09/12 08:06:29.032403 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:06:29.049653 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:10:30.355875 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:10:47.322508 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:14:30.465766 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1

2022/09/12 08:14:31.584362 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1

2022/09/12 08:14:32.326196 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:14:32.327353 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:14:43.899246 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2

2022/09/12 08:14:45.129148 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2

2022/09/12 08:14:50.257699 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:15:16.286823 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:15:44.572108 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:20:39.342677 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1

2022/09/12 08:21:25.467191 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:21:37.830639 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2

2022/09/12 08:21:42.346417 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:21:57.222501 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2

2022/09/12 08:21:58.674256 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:21:59.520835 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:22:10.013196 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2

2022/09/12 08:22:10.184094 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:29:59.792867 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:36:11.287505 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1

2022/09/12 08:36:11.467185 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:36:11.489145 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1

2022/09/12 08:36:21.251414 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:36:21.623667 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 2

2022/09/12 08:36:22.053123 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:36:22.239969 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 2

2022/09/12 08:36:32.238425 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 3

2022/09/12 08:36:32.910382 ERROR RESTY Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 3

2022/09/12 08:36:33.171342 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2

2022/09/12 08:36:33.196758 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2

2022/09/12 08:36:50.847793 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 3

2022/09/12 08:36:52.588487 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 1

2022/09/12 08:37:05.129317 ERROR RESTY context deadline exceeded (Client.Timeout or context cancellation while reading body), Attempt 2`

**Comment by shaojs321** [comment]
同样问题，我是甲骨文部署docker版alist，添加国外的onedrive  pikpak没问题，添加国内的阿里，夸克就有这问题

**Comment by bebestmaple** [comment]
怀疑是哪里有死循环导致的，就算重启docker镜像一段时间后，还是会保持高cpu占用，而后又崩溃。

**Comment by bebestmaple** [comment]
建议beta版本在docker hub中分开标签。我是使用了自动更新docker镜像版本的，目的是能使用最新的稳定版本，而不是beta版，但是如果把beta版设置为lateast，可能不太好。
